src/locales/en/en.json

```diff
@@ -142,6 +142,17 @@
   "COMMON_USERS":"User",
   "COMMON_YES":"Yes",
   "COMMON_NO":"No",
+  "LABEL_HOME":"Home",
+  
+	"ADD_AUTH": "Add Auth Token",
+	"ADD_USERNAME": "Enter Username",
```

Comment:
add screenanme/entity-name as prefix to all lang strings
 @siddhishinde0723 

eg: ADD_USERNAME_INVALID => LOGIN_ADD_USERNAME_INVALID